<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewWidget</class>
 <widget class="QWidget" name="ViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>388</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="QLabel_ViewID">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>View N:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="QLabel_ViewName">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>View Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="OnOff_RenderView">
        <property name="toolTip">
         <string>Enable/disable render this view</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_26" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_26">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QgsMapLayerComboBox" name="QCBox_RenderFile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="QCBox_browseRenderFile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>76</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="RenderWidget" name="render_widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Scale factor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="scaleFactor">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Scale factor for the current view</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.001000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="layerProperties">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Open the properties of this layer</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/plugins/AcATaMa/icons/layer_properties.svg</normaloff>:/plugins/AcATaMa/icons/layer_properties.svg</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RenderWidget</class>
   <extends>QWidget</extends>
   <header>AcATaMa.gui.classification_view_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
